src/chapters/control-flow/conditionals.rst

```diff
@@ -0,0 +1,292 @@
+   :linenos:
+
+   if (condition) {
+      statement1
+      statement2
+      ...
+   } else {
+      statement1
+      statement2
+      ...
+   }
+
+.. index:: ! else if
+
+``else if``
+-----------
+
+.. sourcecode:: java
+   :linenos:
+
+   import java.util.Scanner;
+
+   public class ElseIf {
+      public static void main(String args[]) {
+         Scanner in = new Scanner(System.in);
+         System.out.println('enter a grade');
+         int grade = in.nextInt();
+         if (grade < 60) {
+            System.out.println('F');
+         } else if (grade < 70) {
+            System.out.println('D');
+         } else if (grade < 80) {
+            System.out.println('C');
+         } else if (grade < 90) {
+            System.out.println('B');
+         } else {
+            System.out.println('A');
+         }
+      }
+   }
+
+
+.. index:: ! switch, ! case, ! break
+
+``switch`` Statements
+---------------------
+
+Java also supports a **switch** statement that acts something like an ``else if``
+statement under certain conditions, called **cases**. The ``switch`` statement is not used very often, 
+and we generally recommend you avoid using it. It is not as powerful as the ``else if`` 
+model because the ``switch`` variable can only be compared for equality with a very small
+class of types.
+
+Here is a quick example of a ``switch`` statement:
+
+.. sourcecode:: java
+   :linenos:
+
+   import java.util.Scanner;
+
+   public class DayPrinter {
+      public static void main(String[] args) {
+         Scanner in = new Scanner(System.in);
+         System.out.println("Enter an integer: ");
+         int dayNum = in.nextInt();
+
+         String day;
+         switch (dayNum) {
+            case 0:
+               day = "Sunday";
+               break;
+            case 1:
+               day = "Monday";
+               break;
+            case 2:
+               day = "Tuesday";
+               break;
+            case 3:
+               day = "Wednesday";
+               break;
+            case 4:
+               day = "Thursday";
+               break;
+            case 5:
+               day = "Friday";
+               break;
+            case 6:
+               day = "Saturday";
+               break;
+            default:
+               // in this example, this block runs if none of the above blocks match
+               day = "Int does not correspond to a day of the week";
+         }
+         System.out.println(day);
+      }
+   }
+
+In the example above, if the user entered the number ``4``, "Thursday" would be printed. 
+If the user entered the number ``10``, "Int does
+not correspond to a day of the week" would be printed.
+
+.. index:: ! fallthrough
+
+Additionally, if **break statements** are omitted from the individual
+cases on accident, a behavior known as
+`fallthrough <https://en.wikipedia.org/wiki/Switch_statement#Fallthrough>`__
+is carried out. **Fallthrough** can be quite unintuitive, and is only
```

Comment:
The spell checker flagged "unintuitive".  Google and MS Word seem to think it's OK, however.  Maybe replace it with "non-intuitive"?

Reply:
yep, i saw that but its a word so its fine